Document/0x03-Overview.md

```diff
@@ -1,75 +1,86 @@
+
+## Why Do We Need a Mobile Application Security Testing Guide?
 
-The goal of this project is to help people understand the what, why, when, where, and how of testing applications on Android and iOS devices. The project delivers a complete suite of test cases designed to address the OWASP Mobile Top 10, the Mobile App Security Checklist and the Mobile Application Security Verification Standard (MASVS).
+New technology always introduces new security risks, and mobile computing is no exception. Security concerns for mobile apps differ from traditional desktop software in some important ways, including portability and novelty. Modern mobile operating systems are arguably more secure than traditional desktop operating systems, but problems can still appear when we don't carefully consider security during mobile app development. Data storage, inter-app communication, proper usage of cryptographic APIs, and secure network communication are only some of these considerations. 
 
-## Why Does the World Need a Mobile Application Security Testing Guide?
+>The second paragraph that used to be here is kinda repeated below in Local Data Storage, so I removed the duplicate content and just hinted at it, keeping the explanation down below. 
 
-Every new technology introduces new security risks, and mobile computing is no different. Even though modern mobile operating systems like iOS and Android are arguably more secure by design compared to traditional Desktop operating systems, there's still a lot of things that can go wrong when security is not considered during the mobile app development process. Data storage, inter-app communication, proper usage of cryptographic APIs and secure network communication are only some of the aspects that require careful consideration.
+## Key Areas in Mobile Application Security (AppSec)
 
-Security concerns in the mobile app space differ from traditional desktop software in some important ways. Firstly, while not many people opt to carry a desktop tower around in their pocket, doing this with a mobile device is decidedly more common. As a consequence, mobile devices are more readily lost and stolen, so adversaries are more likely to get physical access to a device and access any of the data stored.
+Many mobile application penetration testing tools have a background in network and web app penetration (app pen) testing, a quality that is valuable for mobile app testing. Almost every mobile app talks to a backend service, and those services are prone to the same kinds of attacks we are familiar with in web apps on desktop machines. Mobile apps differ in that there is a smaller attack surface and therefore more security against injection (attackers supply input to a program) and similar attacks. Instead, we must prioritize data protection on the device and the network to increase mobile security. 
```

Comment:
Hi Sjord. The text originally used the term app pen. Doing a google search, app pen may not be a common industry term...but it seems "pen test" is. Wikipedia says even that term is used pretty casually so maybe it's best to avoid those short forms here. I'll remove.